crates/livesplit-auto-splitting/Cargo.toml

```diff
@@ -0,0 +1,17 @@
+[package]
+name = "livesplit-auto-splitting"
+version = "0.1.0"
+authors = ["Christopher Serr <christopher.serr@gmail.com>"]
+edition = "2021"
+
+[dependencies]
+time = "0.3.3"
+anyhow = "1.0.45"
```

Comment:
We likely want `snafu` instead. This was just a little more convenient for prototyping.